File: 2675-find-the-width-of-columns-of-a-grid/2675-find-the-width-of-columns-of-a-grid.py
class Solution:
    def findColumnWidth(self, grid: List[List[int]]) -> List[int]:
        n = len(grid[0])
        ans = [0 for _ in range(n)]
        for r in grid:
            for j in range(n):
                ans[j] = max(ans[j], len(str(r[j])))
        return ans

File: 2675-find-the-width-of-columns-of-a-grid/2675-find-the-width-of-columns-of-a-grid.rs
impl Solution {
    pub fn find_column_width(grid: Vec<Vec<i32>>) -> Vec<i32> {
        let n:usize = grid[0].len();
        let mut ans = vec![0; n];
        for r in grid {
            for j in 0..n {
                ans[j] = ans[j].max(r[j].to_string().len() as i32);
            }
        }
        ans
    }
}